assets/src/edit-story/components/panels/videoAccessibility.js

```diff
@@ -41,9 +41,21 @@ function VideoAccessibilityPanel({ selectedElements, pushUpdate }) {
     'resource',
     DEFAULT_RESOURCE
   );
-  const poster = getCommonValue(selectedElements, 'poster') || resource.poster;
-  const title = getCommonValue(selectedElements, 'title') || resource.title;
-  const alt = getCommonValue(selectedElements, 'alt') || resource.alt;
+
+  const _defaultForUndefined = (s, d) => (s === undefined ? d : s);
```

Comment:
Why is this needed? Doesn't `getCommonValue` already have a param default value? Why can't we use that?